### ISSUE TITLE
Only a couple of icons are working with aspect

### ISSUE THREAD

**Issue by TheAmended** [iabem97/houdini_issues#6]
I installed Aspect with the dropbox link and _very_ few icons are working with aspect such as Snapchat, Instagram, and that is it. Did I do something wrong?

**Comment by MrARM** [comment]
No, the theme only supports that.  Also system icons don't work yet.
